src/main/java/net/sf/jabref/logic/util/io/FileBasedLock.java

```diff
@@ -72,4 +78,25 @@ public static boolean deleteLockFile(File file) {
         }
         return true;
     }
+
+    /**
+     * Check if a lock file exists, and create it if it doesn't.
+     *
+     * @return true if the lock file already existed
+     * @throws IOException if something happens during creation.
+     */
+    public static boolean createLockFile(File file) throws IOException {
+        File lock = new File(file.getPath() + LOCKFILE_SUFFIX);
+        if (lock.exists()) {
+            return true;
+        }
+        try (FileOutputStream out = new FileOutputStream(lock)) {
+            out.write(0);
+            out.close();
```

Comment:
Is the close necessary? When you use a try with resource (Autocloseable), normally all Streams are closed and freed